.github/pull_request_template.md

```diff
@@ -1,23 +1,33 @@
+<!-- NOTE: write any additional comments here -->
+
 This code review checklist is intended to serve as a starting point for the author and reviewer, although it may not be appropriate for all types of changes (e.g. fixing a spelling typo in documentation).  For more in-depth discussion of how we think about code review, please see [Code Review Guidelines](../blob/main/docs/CODE_REVIEW_GUIDELINES.md).
```

Comment:
I've understood that you'd like to make this part a Note. What changed your mind?

Reply:
It, wasn't my intention to change this to note. do yo prefer it that way?

Reply:
I thought that we all agreed on that in [this](https://github.com/zcash/secant-android-wallet/pull/632#issuecomment-1323414512) discussion. And you pasted the code fragment, which seemed like a note too, to me. But it's up to you.